Automattic/simplenote-ios
# Add New 'About' View to the App

The iOS app is the only Simplenote app that doesn't have an about screen yet. I've mirrored how the about screen looks on other apps while trying to preserve a more iOS feel:

![simulator screen shot - ipad air 2 - 2018-07-11 at 14 31 18](https://user-images.githubusercontent.com/789137/42601187-9cb45d5a-8519-11e8-984b-3d5af9f93e87.png)
![simulator screen shot - iphone 7 - 2018-07-11 at 14 32 01](https://user-images.githubusercontent.com/789137/42601188-9ccb6c98-8519-11e8-8510-6fe2f44ac119.png)

**To Test**
* Start the app, navigate to Settings -> About.
* The view should display well on all devices, sizes and rotations.
* All of the links open in Safari when tapped, including `Privacy` and `Terms` links which I used an attributed string for.
* Tapping `Done` should close the view.
